app/javascript/components/physical-storage-form/physical-storage-form.schema.js

```diff
@@ -50,13 +61,34 @@ const createSchema = (emsId, setEmsId) => ({
         isNotEmpty: true,
       },
     },
+    {
+      component: componentTypes.TEXT_FIELD,
+      name: 'edit',
+      id: 'edit',
+      label: 'edit',
+      hideField: true,
+    },
+    {
+      component: componentTypes.CHECKBOX,
+      id: 'edit_authentication',
+      name: 'edit_authentication',
+      label: __('Edit Authentication'),
+      initialValue: false,
+      condition: {
+        when: 'edit',
```

Comment:
Here `edit` is what we pass in create schema?

Reply:
edit is the value of the hidden edit ddf component. if recordId is not null then edit is 'yes'